tables/subject_infections/choi22b-inf.csv

```diff
@@ -0,0 +1,2 @@
+﻿ref_name,subject_name,infection_date_cmp,infection_date,infected_var_name,location,immune_status,severity,section
+Choi22b,4dose_CP,~,2022-03-01,Unknown variant,South Korea,NULL,NULL,NULL

```

Comment:
@zachandfox for `infection_date_cmp` please use '<' (less than), if we use ~, that could mean Omicron BA.1 or BA.2. I think the paper didn't indicate they were infected during omicron spreading time.